pkg/control/synchronizer_test.go

```diff
@@ -165,21 +169,21 @@ type mockIstio struct {
 	GetCall    bool
 }
 
-func (mi *mockIstio) Delete(_ string, _ *v1.DeleteOptions) error {
+func (mi *mockIstio) Delete(ctx context.Context, _ string, _ v1.DeleteOptions) error {
```

Comment:
Linters would return an error as `ctx` is not used.
The same happens in other places.
```suggestion
func (mi *mockIstio) Delete(_ context.Context, _ string, _ v1.DeleteOptions) error {
```